app/models/position.rb

```diff
@@ -11,7 +11,7 @@ class Position < ApplicationRecord
   has_many :applicants, through: :preferences
 
   validates_presence_of :course_code, :openings, :round
```

Comment:
Is it possible to validate_uniqueness here while `:course_code` must be unique within a certain round + session?